projects/packages/videopress/src/client/block-editor/blocks/videopress/components/videopress-uploader/index.js

```diff
@@ -227,16 +252,60 @@ const VideoPressUploader = ( { attributes, setAttributes, noticeUI, noticeOperat
 			if ( getGuidFromVideoUrl( videoUrl ) ) {
 				return onSelectURL( videoUrl );
 			}
-		}
+		} else if ( media.id ) {
```

Comment:
Can `media.id` be `0`?

Reply:
I'd say no. It's a post ID so it's a positive integer. It's either undefined or a positive integer